cs-minibot-platform-src/src/main/java/minibot/BaseHTTPInterface.java

```diff
@@ -389,6 +412,101 @@ public static void main(String[] args) {
                 return false;
             }
         });
+
+        // Examplescript routes
+        get("/example/patrol/run", (req, res) -> {
+            Patrol mbsd = new Patrol( (FourWheelMovement)
+                    BaseStation
+                            .getInstance().getBotManager().getAllTrackedBots()
+                            .iterator().next().getCommandCenter());
+            mbsd.start();
+            return true;
+        });
+
+        get("/example/patrol/addPointToQueue", (req, res) -> {
```

Comment:
for this route and below, add a really short comment explaining the gist of it. people new to the code will have no idea whats going on here.